keyboards/gmmk/pro/pro.c

```diff
@@ -14,3 +14,232 @@
+    { 75, 76, 77 }, // 24, 4, k47
+    { 78, 79, 80 }, // 25, R, k40
+    { 81, 82, 83 }, // 26, F, k42
+    { 84, 85, 86 }, // 27, V, k44
+    { 90, 91, 92 }, // 28, F5, k07
+    { 93, 94, 95 }, // 29, 5, k46
+    { 96, 97, 98 }, // 30, T, k41
+    { 99, 100, 101 }, // 31, G, k43
+    { 102, 103, 104 }, // 32, B, k45
+    { 105, 106, 107 }, // 33, SPACE, k94
+    { 108, 109, 110 }, // 34, F6, k63
+    { 111, 112, 113 }, // 35, 6, k56
+    { 114, 115, 116 }, // 36, Y, k51
+    { 117, 118, 119 }, // 37, H, k53
+    { 120, 121, 122 }, // 38, N, k55
+    { 126, 127, 128 }, // 39, F7, k71
+    { 129, 130, 131 }, // 40, 7, k57
+    { 132, 133, 134 }, // 41, U, k50
+    { 135, 136, 137 }, // 42, J, k52
+    { 138, 139, 140 }, // 43, M, k54
+    { 144, 145, 146 }, // 44, F8, k76
+    { 147, 148, 149 }, // 45, 8, k67
+    { 150, 151, 152 }, // 46, I, k60
+    { 153, 154, 155 }, // 47, K, k62
+    { 156, 157, 158 }, // 48, ,, k64
+    { 159, 160, 161 }, // 49, Alt_R, k95
+    { 162, 163, 164 }, // 50, F9, ka6
+    { 165, 166, 167 }, // 51, 9, k77
+    { 168, 169, 170 }, // 52, O, k70
+    { 171, 172, 173 }, // 53, L, k72
+    { 174, 175, 176 }, // 54, ., k74
+    { 177, 178, 179 }, // 55, FN, k92
+    { 180, 181, 182 }, // 56, F10, ka7
+    { 183, 184, 185 }, // 57, 0, k87
+    { 186, 187, 188 }, // 58, P, k80
+    { 189, 190, 191 }, // 59, ;, k82
+    { 192, 193, 194 }, // 60, ?, k85
+    { 198, 199, 200 }, // 61, F11, ka3
+    { 201, 202, 203 }, // 62, -, k86
+    { 204, 205, 206 }, // 63, [, k81
+    { 207, 208, 209 }, // 64, ", k83
+    { 213, 214, 215 }, // 65, Ct_R, k04
+};
+
+const aw_led g_aw_leds_2[DRIVER_2_LED_TOTAL] = {
+    { 0, 1, 2 }, // 66, F12, ka5
+    { 12, 13, 14 }, // 67, LED, l01
+    { 15, 16, 17 }, // 68, LED, l11
+    { 21, 22, 23 }, // 69, Prt, k97
+    { 30, 31, 32 }, // 70, LED, l02
+    { 33, 34, 35 }, // 71, LED, l12
+    { 39, 40, 41 }, // 72, Del, k65
+    { 48, 49, 50 }, // 73, LED, l03
+    { 51, 52, 53 }, // 74, LED, l13
+    { 57, 58, 59 }, // 75, PgUp, k15
+    { 66, 67, 68 }, // 76, LED, l04
+    { 69, 70, 71 }, // 77, LED, l14
+    { 72, 73, 74 }, // 78, =, k66
+    { 81, 82, 83 }, // 79, Right, k05
+    { 84, 85, 86 }, // 80, LED, l05
+    { 87, 88, 89 }, // 81, LED, l15
+    { 93, 94, 95 }, // 82, End, k75
+    { 102, 103, 104 }, // 83, LED, l06
+    { 105, 106, 107 }, // 84, LED, l16
+    { 108, 109, 110 }, // 85, BSpc, ka1
+    { 111, 112, 113 }, // 86, PgDn, k25
+    { 120, 121, 122 }, // 87, LED, l07
+    { 123, 124, 125 }, // 88, LED, l17
+    { 126, 127, 128 }, // 89, ], k61
+    { 129, 130, 131 }, // 90, Sh_R, k91
+    { 138, 139, 140 }, // 91, LED, l08
+    { 141, 142, 143 }, // 92, LED, l18
+    { 144, 145, 146 }, // 93, \, ka2
+    { 147, 148, 149 }, // 94, Up, k35
+    { 165, 166, 167 }, // 95, Left, k03
+    { 180, 181, 182 }, // 96, Enter, ka4
+    { 183, 184, 185 }, // 97, Down, k73
+};
+
+const aw_led *g_aw_leds[DRIVER_COUNT] = {
+    g_aw_leds_1,
+    g_aw_leds_2
+};
+#endif
```

Comment:
This should really be kept in line with how the ISSI drivers are configured. 

Eg: 
https://github.com/drashna/qmk_firmware/blob/e40ae50babbb8dd09fe50ff232666fd3fd260354/keyboards/gmmk/pro/pro.c#L18-L175

Which would use this instead of the 3 different structures that you're using. 

```c
const aw_led g_aw_leds[DRIVER_LED_TOTAL]
```